CHANGELOG.md

```diff
@@ -3,7 +3,90 @@
+- The discovery plugin no longer panics in Trigger() if downloader is nil
+- The bundle plugin now ignores service errors for file:// resources
+- The bundle plugin file loader was updated to support directories
+- A timer to HTTP request was added to the downloader
+- The requested_by field in the logging plugin is now optional
+
+### Rego
+
+- topdown: Fix confusing error message for minus operator (#4090) ([#1643](https://github.com/open-policy-agent/opa/issues/1643))
+- ast/compile: fix print rewriting for arrays in := and head vars ([#4078](https://github.com/open-policy-agent/opa/issues/4078))
+- ast/compile: fix print rewriting for arrays in unification (#4099) ([#4078](https://github.com/open-policy-agent/opa/issues/4078))
+- ast+topdown: cache Array groundness, use for plugging (#4151) ([#3679](https://github.com/open-policy-agent/opa/issues/3679))
+- ast/compile: check arity in undefined function stage (#4059) ([#4054](https://github.com/open-policy-agent/opa/issues/4054))
+
+### SDK
+
+- The `opa.runtime()` built-in is now made available to the SDK ([#4050](https://github.com/open-policy-agent/opa/issues/4050) authored by @[oren-zohar](https://github.com/oren-zohar) and @[cmschuetz](https://github.com/cmschuetz)
+- Plugins are now exposed on the SDK object
+- The SDK now supports graceful shutdown (#4119) ([#3980](https://github.com/open-policy-agent/opa/issues/3980)) reported by @[brianchhun-chime](https://github.com/brianchhun-chime)
+- `print` output is now sent to the configured logger
+
+### Docs
+
+- All pages in the docs now have a feedback button (#4096) ([#3664](https://github.com/open-policy-agent/opa/issues/3664)) authored by @[alan-ma](https://github.com/alan-ma)
+- The Kafka docs have been updated to use the new Kafka plugin, and to use the OPA management APIs
+- The Terraform tutorial was updated to use `opa exec` ([#3965](https://github.com/open-policy-agent/opa/issues/3965))
+- The docs on Contributing as well as the Vendor Guidelines have been updated
+- The term "whitelist" has been replaced by "allowlist" across the docs
+- A simple destructuring assignment example was added to the docs
+- The docs have been reviewed on the use of assignment, equality and comparison operators, to make sure they follow best practice
+
+### CI
+
+- SHA256 checksums of CI builds now published to release directory ([#3448](https://github.com/open-policy-agent/opa/issues/3448)) authored by @[johanneslarsson](https://github.com/johanneslarsson) reported by @[raesene](https://github.com/raesene)
+- golangci-lint upgraded to v1.43.0 (#4173) (authored by @[shuheiktgw](https://github.com/shuheiktgw))
+- The build now creates an executable for darwin/arm64 (experimental)
+- PRs targeting the [ecosystem](https://www.openpolicyagent.org/docs/latest/ecosystem/) page are now checked for mistakes using Rego policies
+
+### Wasm
+
+- wasm: Update generated binaries (authored by @[](https://github.com/))
```

Comment:
We can drop this. It's been a side-effect of the reverse  builtin being implemented natively, too. We could mention that in the bullet point for that new built-in.